app/src/main/res/drawable/ic_launcher_background.xml

```diff
@@ -15,14 +15,11 @@
      limitations under the License.
 -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="108dp"
-    android:height="108dp"
-    android:viewportWidth="108"
-    android:viewportHeight="108">
-  <group android:scaleX="0.99722993"
-      android:scaleY="0.99722993">
-      <path
-          android:pathData="M26.04,0L82.26,0A26.04,26.04 0,0 1,108.3 26.04L108.3,82.26A26.04,26.04 0,0 1,82.26 108.3L26.04,108.3A26.04,26.04 0,0 1,0 82.26L0,26.04A26.04,26.04 0,0 1,26.04 0z"
-          android:fillColor="#0097a7"/>
-  </group>
-</vector>
\ No newline at end of file
+    android:width="108.3dp"
```

Comment:
why is the change from 108 to 108.3? Is this the resouce in starter code? If not is there a step in the codelab to perform this change.

Reply:
This was built off activities branch. Also says 108.3 in starter.